Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -2102,6 +2102,26 @@ final class MigrationTests: XCTestCase {
         // OrderItem's addOns relationship exists.
         XCTAssertEqual(migratedOrderItem.value(forKey: "addOns") as? NSOrderedSet, [addOn])
     }
+
+    func test_migrating_from_94_to_95_enables_creating_new_TaxRate_entity() throws {
+        // Given
+        let sourceContainer = try startPersistentContainer("Model 94")
+        let sourceContext = sourceContainer.viewContext
+
+        try sourceContext.save()
```

Comment:
Just because it's a migration I think we could add extra assertions, like:
```suggestion
        try sourceContext.save()
        
        // Confidence Check. This entity should not exist in Model 94.
        XCTAssertNil(NSEntityDescription.entity(forEntityName: "TaxRate", in: sourceContext))
```